content/zh/docs/installing-on-linux/introduction/air-gapped-installation.md

```diff
@@ -7,657 +7,554 @@ linkTitle: "离线安装"
-- 建议您事先创建一个操作系统镜像文件，并且安装好所有相关依赖项。这样，您便可以直接使用该镜像文件在每台机器上安装操作系统，提高部署效率，也不用担心任何依赖项问题。
-
-{{</ notice >}} 
-
-### 容器运行时
-
-您的集群必须有一个可用的容器运行时。在离线环境中创建集群之前，您必须手动安装 Docker 或其他容器运行时。
-
-{{< content "common/container-runtime-requirements" >}}
-
-### 网络和 DNS 要求
-
-{{< content "common/network-requirements" >}}
-
-### 示例机器
-
-本示例包含三台主机，如下所示，主节点充当任务机。
-
-| 主机 IP     | 主机名称 | 角色         |
-| ----------- | -------- | ------------ |
-| 192.168.0.2 | master   | master, etcd |
-| 192.168.0.3 | node1    | worker       |
-| 192.168.0.4 | node2    | worker       |
-
-## 步骤 2：准备一个私有镜像仓库
+   ```bash
+   curl -sfL https://get-kk.kubesphere.io | VERSION=v2.0.0 sh -
+   ```
 
-您可以使用 Harbor 或者其他任意私有镜像仓库。本教程以 Docker 仓库作为示例，并使用[自签名证书](https://docs.docker.com/registry/insecure/#use-self-signed-certificates)（如果您有自己的私有镜像仓库，可以跳过这一步）。
+   {{</ tab >}}
 
-### 使用自签名证书
+   {{< tab "如果您访问 GitHub/Googleapis 受限" >}}
 
-1. 执行以下命令生成您自己的证书：
+   首先运行以下命令，以确保您从正确的区域下载 KubeKey。
 
    ```bash
-   mkdir -p certs
+   export KKZONE=cn
    ```
 
+   运行以下命令来下载 KubeKey：
+
    ```bash
-   openssl req \
-   -newkey rsa:4096 -nodes -sha256 -keyout certs/domain.key \
-   -x509 -days 36500 -out certs/domain.crt
+   curl -sfL https://get-kk.kubesphere.io | VERSION=v2.0.0 sh -
    ```
+   {{</ tab >}}
 
-2. 当您生成自己的证书时，请确保在字段 `Common Name` 中指定一个域名。例如，本示例中该字段被指定为 `dockerhub.kubekey.local`。
-
-   ![自签名证书](/images/docs/zh-cn/installing-on-linux/introduction/air-gapped-installation/self-signed-cert.jpg)
+   {{</ tabs >}}
 
-### 启动 Docker 仓库
+2. 在源集群中使用 KubeKey 创建 manifest。支持下面 2 种方式：
 
-执行以下命令启动 Docker 仓库：
-
-```
-docker run -d \
-  --restart=always \
-  --name registry \
-  -v "$(pwd)"/certs:/certs \
-  -v /mnt/registry:/var/lib/registry \
-  -e REGISTRY_HTTP_ADDR=0.0.0.0:443 \
-  -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/domain.crt \
-  -e REGISTRY_HTTP_TLS_KEY=/certs/domain.key \
-  -p 443:443 \
-  registry:2
-```
-
-### 配置仓库
-
-1. 在 `/etc/hosts` 中添加一个条目，将主机名（即仓库域名；在本示例中是 `dockerhub.kubekey.local`）映射到您机器的私有 IP 地址，如下所示。
+   - （推荐）在已创建的集群中执行 KubeKey 命令生成该文件。
 
    ```bash
-   # docker registry
-   192.168.0.2 dockerhub.kubekey.local
+   ./kk create manifest
    ```
 
-2. 执行以下命令，复制证书到指定目录，并使 Docker 信任该证书。
+   - 根据模版手动创建并编写该文件。关于更多信息，请参阅 [manifest-example](https://github.com/kubesphere/kubekey/blob/master/docs/manifest-example.md)。
 
+3. 执行以下命令在源集群中修改 manifest 配置：
+   
    ```bash
-   mkdir -p  /etc/docker/certs.d/dockerhub.kubekey.local
+   vim manifest.yaml
    ```
-
-   ```bash
-   cp certs/domain.crt  /etc/docker/certs.d/dockerhub.kubekey.local/ca.crt
+   
+   ```yaml
+   ---
+   apiVersion: kubekey.kubesphere.io/v1alpha2
+   kind: Manifest
+   metadata:
+     name: sample
+   spec:
+     arches:
+     - amd64
+     operatingSystems:
+     - arch: amd64
+       type: linux
+       id: centos
+       version: "7"
+       repository:
+         iso:
+           localPath: ""
+           url: #https://github.com/kubesphere/kubekey/releases/download/v2.0.0/centos-7-amd64-rpms.iso
```

Comment:
url: "https://github.com/kubesphere/kubekey/releases/download/v2.0.0/centos-7-amd64-rpms.iso"

Reply:
If users are not able to download the iso file from the GitHub release (especially in CN), it there any other options?

Reply:
> If users are not able to download the iso file from the GitHub release (especially in CN), are there any other options?

Users can make their own system dependency packages, which can be specified by `localPath`.